tests/themis/themis_secure_cell.c

```diff
@@ -366,10 +366,261 @@ static void secure_cell_api_test(void)
 	secure_cell_api_test_context_imprint();
 }
 
+/*
+ * See themis_auth_sym_message_hdr_t definition.
+ */
+static inline uint32_t* context_iv_length(uint8_t* context)
+{
+	return (uint32_t*)context + 1;
+}
+static inline uint32_t* context_auth_tag_length(uint8_t* context)
+{
+	return (uint32_t*)context + 2;
+}
+static inline uint32_t* context_message_length(uint8_t* context)
+{
+	return (uint32_t*)context + 3;
+}
```

Comment:
these function names are counterintuitive: I would assume they return integers, not pointers. But in your code you seem to need pointers to modify data, so I would call them something like `ctx_iv_len_pos` (as in position)

Reply:
I prefer to use getter/setter which returns value (not pointer) and set value to correct field of struct to change. without needs to operate with pointers and readable api : )
```
uint32_t iv_length = get_context_iv_length(context);
set_context_iv_length(context, new_length)
```

Reply:
I've updated the code to use getters and setters. It really looks nicer now. Thanks, @Lagovas @secumod.